src/components/SiteDetail.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import summary from '../apis/summary';
+class SiteDetail extends Component {
+  state = {
+    loadedSites: null
+  };
+
+  componentDidMount() {
+    this.loadSiteData();
+  }
+
+  componentDidUpdate() {
+    this.loadSiteData();
+  }
+
+  loadSiteData() {
+    if (this.props.match.params.id) {
+      if (
+        !this.state.loadedSites ||
+        (this.state.loadedSites && this.state.loadedSites.id !== this.props.match.params.id)
+      ) {
+        summary.get('/sites/' + this.props.match.params.id).then(response => {
+          this.setState({ loadedSites: response.data });
+        });
+      }
+    }
+  }
+  render() {
+    return (
+      <div>
+        <section>Site Detail View Page</section>
+      </div>
```

Comment:
Bold move, having a section within a div? do you need the divs? Do they do anything special? 

If not, you can always slum it and use React Fragments!

```js
  return (
   <>
      <section> site detail view page </section>
   </>
  )
```

Reply:
Or, you could just use the sections. You certainly don't need the section-in-a-div.